simple_example.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
If that's ok, I'd rather keep this kind of sequence in the instructions first. I'm working on creating pipelines based on each of the scripts, we could use a more consistent and configurable approach for running the whole thing locally.